requests/models.py

```diff
@@ -447,9 +448,14 @@ def prepare_headers(self, headers):
         self.headers = CaseInsensitiveDict()
         if headers:
             for header in headers.items():
+                name, value = header
+                if value is None:
+                    if hame.lower() in {"user-agent", "accept-encoding", "host"}:
```

Comment:
Can use `urllib3.util.SKIPPABLE_HEADERS` here. Also typo `hame` -> `name`

Reply:
Thanks for the pointer on the constant and a silly typo